packages/fhir/src/Adaptor.js

```diff
@@ -3,6 +3,15 @@ import { expandReferences } from '@openfn/language-common/util';
 
 import { handleResponse, request } from './Utils';
 
+/**
+ * Options provided to the Create request
```

Comment:
All http methood (get create post) all support these options - so we can't say Create here

Reply:
Ok, this documentation is basically all wrong. It's half wrong now and it's half wrong when we make our changes.

We don't have time to fix it.

For this release, I want the documentation to be vague.

Please remove this type def. I will add light documentation to `post`, but we should basically describe the `params` type as an object